src/supermarket/app/controllers/cookbooks_controller.rb

```diff
@@ -31,7 +31,12 @@ def index
       @cookbooks = @cookbooks.featured
     end
 
-    if @current_params[:order].present?
+    if (order = @current_params[:order]).present?
+
+      if order == "recently_updated"
+        @cookbooks = @cookbooks.undeprecated
```

Comment:
"undeprecated" sounds wrong. A cookbook is either deprecated, or it is not deprecated.

Can we `@cookbooks = @cookbooks - @cookbooks.deprecated` ?

Or change the scope to something like: `scope :not_deprecated, -> { where.not(id: deprecated) }` ?